source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -33,7 +33,7 @@ tests:
       - name: commitTransaction
         object: session0
         result:
-          errorLabelsOmit: ["RetryableWriteError", "TransientTransactionError", "UnknownTransactionCommitResult"]
+          errorLabelsOmit: ["RetryableWriteError", "TransientTransactionError"]
```

Comment:
Although its not retryable I believe it still can be marked as `UnknownTransactionCommitResult`

Reply:
Yes that sounds correct to me. Interesting to note is that although session.commit_transaction() will not retry, withTransaction will.